src/features/slop/hooks/useMapPinch.ts

```diff
@@ -46,7 +44,13 @@ const useMapPinch = (containerRef: RefObject<HTMLElement>) => {
             height: containerRef.current!.getBoundingClientRect().height,
           }
         );
-        api.start({ x: boundedX, y: boundedY });
+
+        if (x < minX) x = minX;
+        if (x > maxX) x = maxX;
```

Comment:
```suggestion
        const boundedX = Math.min(Math.max(x, min), max)
```
가져온 x 값을 직접 바꾸는 것보다 이런 식으로 새로운 변수에 넣어주는게 좋을 것 같아!